examples/stable_baselines3/argument_parser.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+
+
+def default_argument_parser(program: str = None):
```

Comment:
Lets keep the examples as lean as possible. This `argument_parser.py` file can be removed. 

I think the options: 
+ `scenarios`: Move this to the `config.yaml` file. For example, see pr #1141 .
+ `--sim-name`: Just give a fixed value.
+ `--headless`: This is already moved to `run.py` file.
+ `--seed`: No need for options. Just use default value.
+ `--sumo-port`: No need for options. Just use default value.
+ `--episodes`: If used for training/evaluation, provide default values and move it to `config.yaml`.